lib/siwapp_web/live/invoices_live/index.ex

```diff
@@ -0,0 +1,42 @@
+defmodule SiwappWeb.InvoicesLive.Index do
+  use SiwappWeb, :live_view
+  alias Siwapp.Invoices
+
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(:invoices, Invoices.list())
+     |> assign(:checked, MapSet.new())}
+  end
+
+  def handle_event("click_checkbox", data, socket) do
+    checked = update_checked(data, socket)
+    IO.inspect(checked)
```

Comment:
I see you don't check your PRs before me. Please, ALWAYS do that, and don't tell me I forgot to delete that. 
What you forgot is to see the changes you want to merge.